common/src/main/res/layout/activity_replay.xml

```diff
@@ -0,0 +1,446 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Layouts for replay and teaser are more different now, decided to add separate layout for now, since soon we'll unite all three apps together.